src/libunwind.zig

```diff
@@ -45,7 +45,7 @@ pub fn buildStaticLib(comp: *Compilation, prog_node: *std.Progress.Node) !void {
                 try cflags.appendSlice(&[_][]const u8{
                     "-fno-rtti",
                     "-I",
-                    try comp.zig_lib_directory.join(arena, &[_][]const u8{ "libcxx", "include" }),
+                    try comp.zig_lib_directory.join(arena, &[_][]const u8{ "include" }),
```

Comment:
The patch I was thinking of was more like this:

```suggestion
                    "-nostdinc++",
```

Reply:
Zig unconditionally adds `-nostdinc++`:

https://github.com/ziglang/zig/blob/9e19969e09ffee8aca39f28a42d566cc62b479d6/src/Compilation.zig#L4257-L4259

Reply:
So removing line libunwind.zig:48 completely should do the trick?

Reply:
force pushed the change removing -I... completely